cli/package.json

```diff
@@ -24,18 +26,27 @@
   },
   "dependencies": {
     "@commander-js/extra-typings": "^12.1.0",
+    "@listr2/prompt-adapter-enquirer": "^2.0.11",
     "@massalabs/massa-web3": "^5.0.0-next",
-    "commander": "^12.1.0"
+    "commander": "^12.1.0",
+    "js-sha256": "^0.11.0",
+    "listr2": "^8.2.4",
+    "yaml": "^2.5.1"
   },
   "devDependencies": {
+    "@eslint/eslintrc": "^3.1.0",
+    "@eslint/js": "^9.10.0",
     "@massalabs/eslint-config": "^0.0.11",
-    "@types/jest": "^29.5.12",
+    "@types/jest": "^29.5.13",
     "@types/node": "^22.5.3",
-    "@typescript-eslint/eslint-plugin": "^8.4.0",
-    "eslint": "^8.6.0",
+    "@typescript-eslint/eslint-plugin": "^8.6.0",
+    "eslint": "^9.10.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-jsdoc": "^50.2.2",
+    "jest": "^29.7.0",
     "prettier": "^3.3.3",
+    "ts-jest": "^29.2.5",
+    "ts-node": "^10.9.2",
```

Comment:
We czan remove ts-node and use tsx

Reply:
we discussed about it, we will keep ts-node for now as it's enough for now
we might even remove ts-node to directly build and run the cli when using `dev` script